examples/Migration/CampaignManagement/CreateCompleteCampaignBothApisPhase1.php

```diff
@@ -0,0 +1,424 @@
+ *
+ * In this code example, the functionality to create a campaign budget
+ * has been migrated to the Google Ads API. The remaining functionalities - creating a campaign,
+ * creating an ad group, creating expanded text ads and creating keywords - are using the AdWords
+ * API.
+ */
+class CreateCompleteCampaignBothApisPhase1
+{
+
+    // Number of ads being added/updated in this code example.
+    private const NUMBER_OF_ADS = 5;
+    // The list of keywords being added in this code example.
+    private const KEYWORDS_TO_ADD = [
+        "mars cruise",
+        "space hotel"
+    ];
+
+    // The default page size for search queries.
+    const PAGE_SIZE = 1000;
+
+    /**
+     * Runs the CreateCompleteCampaignBothApisPhase1 example.
+     * @param AdWordsServices $adWordsServices the AdWords services
+     * @param AdWordsSession $adWordsSession the AdWords session
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param string $customerId the client customer ID without hyphens
+     */
+    public static function runExample(
+        AdWordsServices $adWordsServices,
+        AdWordsSession $adWordsSession,
+        GoogleAdsClient $googleAdsClient,
+        string $customerId
+    ) {
+        $campaignBudget = self::createCampaignBudget($googleAdsClient, $customerId);
+        $campaign = self::createCampaign(
+            $adWordsServices,
+            $adWordsSession,
+            $campaignBudget->getId()->getValue()
+        );
+        $adGroup = self::createAdGroup(
+            $adWordsServices,
+            $adWordsSession,
+            $campaign->getId()
+        );
+        self::createTextAds($adWordsServices, $adWordsSession, $adGroup->getId());
+        self::createKeywords(
+            $adWordsServices,
+            $adWordsSession,
+            $adGroup->getId(),
+            self::KEYWORDS_TO_ADD
+        );
+    }
+
+    /**
+     * Creates a campaign budget.
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param string $customerId the client customer ID without hyphens
+     * @return CampaignBudget the newly created campaign budget
+     */
+    private static function createCampaignBudget(
+        GoogleAdsClient $googleAdsClient,
+        string $customerId
+    ) {
+        $campaignBudgetServiceClient = $googleAdsClient->getCampaignBudgetServiceClient();
+
+        // Creates a campaign budget.
+        $campaignBudget = new CampaignBudget([
+            'name' => new StringValue(['value' => 'Interplanetary Cruise Budget #' . uniqid()]),
+            'delivery_method' => BudgetDeliveryMethod::STANDARD,
+            'amount_micros' => new Int64Value(['value' => 500000])
+        ]);
+
+        // Creates a campaign budget operation.
+        $campaignBudgetOperation = new CampaignBudgetOperation();
+        $campaignBudgetOperation->setCreate($campaignBudget);
+
+        /** @var MutateCampaignBudgetResponse $campaignBudgetResponse */
```

Comment:
You need to do import of MutateCampaignBudgetResponse

Reply:
Same for the same place of other files.

Reply:
Removed the @var declaration, as it's not being used in other sections of the example as well.